doc/design/wallet/wallet-design.md

```diff
@@ -320,7 +356,8 @@ pub extern fn indy_register_wallet_storage(command_handle: i32,
                                            add_record: Option<extern fn(storage_handle: i32,
                                                                         type_: *const c_char,
                                                                         id: *const c_char,
-                                                                        value: *const c_char,
+                                                                        value: *const u8,
```

Comment:
For value, it is used sometimes *const u8 and sometimes *const c_char. I would guess both will work but we should chose some.